### PR TITLE
Update "Manage resource assignments" table#24038

### PR DESCRIPTION
**Pull request by jeswanke**
stolostron/backlog#24038

Signed-off-by: John Swanke <jswanke@redhat.com>

### CODE REVIEW

**Comment by KevinFCormier** [comment]
@jeswanke Some issues with this... the table inside the modal doesn't really fit and needs to include the column that states whether the cluster is being added or removed.
image

There is a broken test case as well. I think we might want to examine a bunch of the other strings and update them as well. There are still many references to "resources" where it could just say clusters.

**Comment by jeswanke** [comment]
table now shows change

image

File: frontend/src/routes/Infrastructure/Clusters/ClusterSets/ClusterSetDetails/ClusterSetManageResources/ClusterSetManageResources.tsx [hunk 1]
```diff
@@ -67,39 +76,95 @@ export function ClusterSetManageResourcesContent() {
     const deploymentDictionary = new Map<string | undefined, ClusterDeployment>()
     clusterDeployments?.forEach((deployment) => deploymentDictionary.set(deployment.metadata.name, deployment))
 
-    const [managedClusters, managedClusterSets] = useRecoilValue(
-        waitForAll([managedClustersState, managedClusterSetsState])
-    )
+    const clusters = useAllClusters()
+    const [managedClusterSets] = useRecoilValue(waitForAll([managedClustersState, managedClusterSetsState]))
+    const [clusterCurators] = useRecoilState(clusterCuratorsState)
+
```
Comment (KevinFCormier) [inline frontend/src/routes/Infrastructure/Clusters/ClusterSets/ClusterSetDetails/ClusterSetManageResources/ClusterSetManageResources.tsx:82]: I'm not sure if `waitForAll` was ever needed... certainly looks like it shouldn't be now. What do you think?

File: frontend/src/routes/Infrastructure/Clusters/ClusterSets/ClusterSetDetails/ClusterSetManageResources/ClusterSetManageResources.tsx [hunk 2]
```diff
@@ -67,39 +76,95 @@ export function ClusterSetManageResourcesContent() {
     const deploymentDictionary = new Map<string | undefined, ClusterDeployment>()
     clusterDeployments?.forEach((deployment) => deploymentDictionary.set(deployment.metadata.name, deployment))
 
-    const [managedClusters, managedClusterSets] = useRecoilValue(
-        waitForAll([managedClustersState, managedClusterSetsState])
-    )
+    const clusters = useAllClusters()
+    const [managedClusterSets] = useRecoilValue([managedClustersState, managedClusterSetsState])
```
Comment (KevinFCormier) [inline frontend/src/routes/Infrastructure/Clusters/ClusterSets/ClusterSetDetails/ClusterSetManageResources/ClusterSetManageResources.tsx:80]:
```suggestion
    const managedClusterSets = useRecoilValue(managedClusterSetsState)
```

**Comment by KevinFCormier** [comment]
Superseded by https://github.com/stolostron/console/pull/1888

**Pull request closed without merging**